projects/1.21/assets/more-sniffer-flowers/moresnifferflowers/lang/zh_cn.json

```
@@ -54,13 +101,66 @@
 	"block.moresnifferflowers.giant_beetroot": "巨型甜菜根",
 	"block.moresnifferflowers.rebrewing_stand": "复酿台",
 	"block.moresnifferflowers.dyespria_plant": "染瓣兰",
+	"block.moresnifferflowers.dyescrapia_plant": "削色兰",
+	"block.moresnifferflowers.vivicus_log": "芽生原木",
+	"block.moresnifferflowers.vivicus_wood": "芽生木",
+	"block.moresnifferflowers.stripped_vivicus_log": "去皮芽生原木",
+	"block.moresnifferflowers.stripped_vivicus_wood": "去皮芽生木",
+	"block.moresnifferflowers.vivicus_planks": "芽生木板",
+	"block.moresnifferflowers.vivicus_stairs": "芽生木楼梯",
+	"block.moresnifferflowers.vivicus_slab": "芽生木台阶",
+	"block.moresnifferflowers.vivicus_fence": "芽生木栅栏",
+	"block.moresnifferflowers.vivicus_fence_gate": "芽生木栅栏门",
+	"block.moresnifferflowers.vivicus_door": "芽生木门",
+	"block.moresnifferflowers.vivicus_trapdoor": "芽生木活板门",
+	"block.moresnifferflowers.vivicus_pressure_plate": "芽生木压力板",
+	"block.moresnifferflowers.vivicus_button": "芽生木按钮",
+	"block.moresnifferflowers.vivicus_leaves": "芽生树叶",
+	"block.moresnifferflowers.vivicus_sapling": "芽生树苗",
+	"block.moresnifferflowers.vivicus_sign": "芽生木告示牌",
+	"block.moresnifferflowers.vivicus_hanging_sign": "悬挂式芽生木告示牌",
+	"block.moresnifferflowers.sprouting_vivicus_leaves": "抽芽的芽生树叶",
+	"block.moresnifferflowers.vivicus_leaves_sprout": "芽生树叶芽",
+	"block.moresnifferflowers.corrupted_sprouting_vivicus_leaves": "抽芽的腐坏芽生树叶",
+	
+	"block.moresnifferflowers.corrupted_log": "腐坏原木",
+	"block.moresnifferflowers.corrupted_wood": "腐坏木",
+	"block.moresnifferflowers.stripped_corrupted_log": "去皮腐坏原木",
+	"block.moresnifferflowers.stripped_corrupted_wood": "去皮腐坏木",
+	"block.moresnifferflowers.corrupted_planks": "腐坏木板",
+	"block.moresnifferflowers.corrupted_stairs": "腐坏木楼梯",
+	"block.moresnifferflowers.corrupted_slab": "腐坏木台阶",
+	"block.moresnifferflowers.corrupted_fence": "腐坏木栅栏",
+	"block.moresnifferflowers.corrupted_fence_gate": "腐坏木栅栏门",
+	"block.moresnifferflowers.corrupted_door": "腐坏木门",
+	"block.moresnifferflowers.corrupted_trapdoor": "腐坏木活板门",
+	"block.moresnifferflowers.corrupted_pressure_plate": "腐坏木压力板",
+	"block.moresnifferflowers.corrupted_button": "腐坏木按钮",
+	"block.moresnifferflowers.corrupted_leaves": "腐坏树叶",
+	"block.moresnifferflowers.corrupted_leaves_bush": "腐坏树叶丛",
+	"block.moresnifferflowers.corrupted_sapling": "腐坏树苗",
+	"block.moresnifferflowers.corrupted_sign": "腐坏木告示牌",
+	"block.moresnifferflowers.corrupted_hanging_sign": "悬挂式腐坏木告示牌",
+	"block.moresnifferflowers.corrupted_sludge": "腐坏秽浆",
+	"block.moresnifferflowers.corrupted_slime_layer": "腐坏黏液层",
+
+	"block.moresnifferflowers.decayed_log": "腐朽原木",

```

decayed和corrupted需要区分吗？感觉腐朽和腐坏有点接近了

---

需要，Corrupted是树种的木材，这个是把上面的腐坏黏液球投到其他原木后形成的方块
```suggestion
	"block.moresnifferflowers.decayed_log": "朽烂的原木",
```